import-export-cli/integration/apim/apim.go

```diff
@@ -608,6 +608,63 @@ func (instance *Client) AddAPIProductFromJSON(t *testing.T, path string, usernam
 	return apiProductResponse.ID
 }
 
+// CreateAPIRevision : Create API revision
+func (instance *Client) CreateAPIRevision(apiID string) *APIRevision {
+	apiProductsURL := instance.publisherRestURL + "/apis/" + apiID + "/revisions"
```

Comment:
Lets change `apiProductsURL` to something generic like `revisioningURL`

Reply:
I think I have made the same mistake in CreateAPIProductRevision(), please fix it for me :)

Reply:
Sure. Fixed via https://github.com/wso2/product-apim-tooling/pull/639/commits/9575af27ae7dcd8620fac2dba417df0c30eec04a